### ISSUE TITLE
ip2c API very slow response

### ISSUE THREAD

**Issue by Mkaddem97** [yogakurniawan/vue-tel-input-vuetify#56]
The ip2c API is way too slow.
Could you please make the API url changeable?

**Comment by redvivi** [comment]
Same here, facing 1000ms response time.

**Comment by Rpierillas** [comment]
Yes please :) This would be a welcome change, and sending the IP address to some external API could make RGPD breach on some application, being able to change the API url would be a very needed change ;) 

If you need to catch the user's country, it might be better to do it based on the browser's locale. No breach, and no call to external API ;)
